### PR TITLE
Add a welcome message for first merge

### PR DESCRIPTION
**Pull request by pelson**
Uses https://github.com/apps/welcome to give a friendly message on merged PRs for first-time contributors.

I don't have confidence in the multi-line string and emojis... I'd probably want to test this out on a fresh repo somewhere before enabling.

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: >
+    Congratulations on having your first pull request merged!
+    The Iris community are stoked that you've contributed and we all look forward to
+    your future work!
```
Comment (dkillick) [inline .github/config.yml]: @pelson nice 👍 I wonder whether we don't need this second exclamation mark though – it almost makes the cheer feel forced. I'd personally go for a smile 😄 here, but a full stop would also work.

Comment (pelson) [reply .github/config.yml]: Good suggestion. Will test this out on a new repo (I'll grab you to come help me out if you don't mind) and then we can merge.

**Comment by pelson** [comment]
Done and done. Thanks @dkillick. I've enabled this as an app for Iris now (we can grow it to the wider scitools org as and when we want). Now to find somebody to test this out on... 😉

**Comment by pelson** [comment]
@dkillick and I tried this in https://github.com/pelson/repohealth.info/pull/22 with no success...

Closing this until we have a working proof-of-concept.

**Pull request closed without merging**